doc/nrf/libraries/fem_al/index.rst

```diff
@@ -0,0 +1,48 @@
+.. _fem_al_lib:
```

Comment:
If you want to add this lib doc to the Other libraries section, place it in doc/nrf/libraries/other and rename this file to fem_al.rst, for example